core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -106,6 +106,9 @@ class SparkSubmitSuite
   // Necessary to make ScalaTest 3.x interrupt a thread on the JVM like ScalaTest 2.2.x
   implicit val defaultSignaler: Signaler = ThreadSignaler
 
+  private val emptyIvySettings = File.createTempFile("ivy", ".xml")
```

Comment:
@vanzin, do we need to delete this back?

Reply:
It's not necessary to delete temp files in tests. They'll go away when you do a "clean".